README.md

```diff
@@ -7,3 +8,129 @@
 A dataframe auditor that computes a number characteristics of the data. This is done by profiling the frame.
```

Comment:
"Profiling the frame". Rather say "dataframe"

Reply:
removed `A dataframe auditor that computes a number characteristics of the data. This is done by profiling the frame. `